src/platform/w3dengine/client/w3dstatuscircle.cpp

```diff
@@ -0,0 +1,290 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief W3D status circle
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "w3dstatuscircle.h"
+#include "dx8indexbuffer.h"
+#include "dx8vertexbuffer.h"
+#include "dx8wrapper.h"
+#include "gamelogic.h"
+#include "globaldata.h"
+#include "scriptengine.h"
+#include "vertmaterial.h"
+
+int W3DStatusCircle::m_diffuse = 0xFF;
+bool W3DStatusCircle::m_needUpdate;
+
+W3DStatusCircle::W3DStatusCircle() :

```

Comment:
add m_numTriangles init to 0, comment as bugfix

Reply:
fixed.